soccer/Robot.hpp

```diff
@@ -548,6 +548,7 @@ class OurRobot : public Robot {
     std::shared_ptr<Geometry2d::Circle> createBallObstacle() const;
 
     friend class Processor;
+    friend class RadioNode;
```

Comment:
I don't think the `friend class Processor` is needed anymore

Reply:
I believe it is, because Processor.cpp makes call to protected objects in the Robot class, such as angleFunctionPath.